python/tests/test_combinatorics.py

```diff
@@ -1335,6 +1340,10 @@ def test_branch_length_semantics(self):
         length = time[edges.parent] - time[edges.child]
         assert np.all(length == branch_length)
 
+    def test_kwargs(self):
```

Comment:
Can put this into TreeGeneratorTestBase class, and use ``self.method(3, tracked_samples=[0, 1])`` to automatically test for all the generator methods.

Reply:
Ah, I thought about this but wasn't sure how to do it. Thanks for the tip.

Reply:
Done